clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/AbstractStateModelFactory.java

```diff
@@ -53,6 +57,10 @@ public ExecutorService getExecutorService(String resourceName) {
     return executorService;
   }
 
+  public ParticipantStateTransitionStats getStateTransitionStats(String resourceName) {
```

Comment:
Why do you need `resourceName` param since it is not being used?

Reply:
I realized that this is an overrided function, so can you put the right annotation here?

Reply:
This is the root of this function actually; I can't add annotation here; but the annotation was added in `LeaderFollowerPartitionStateModelDualPoolFactory` which indeed override it.

I added a javaDoc in the latest diff to explain the input.

Reply:
I see.